tests/openapi/test_openapi.py

```diff
@@ -203,11 +203,32 @@ def setUp(self):
             "paths": {
                 "/foo": {"post": {"a": [1, 2, "b"], "responses": {"something": "is already here"}}},
                 "/bar": {"get": {_X_INTEGRATION: {"a": "b"}}},
+                "/nullmethod": {"get": None},
             },
         }
 
         self.editor = OpenApiEditor(self.original_openapi)
 
+    def test_must_override_null_path(self):
+        path = "/newpath"
```

Comment:
could you explain what you are trying to test here. my understanding is you are testing that the `/nullmethod` path in the `original_openapi` defined in the `setUp` method  will be override. But I am not able to understand why you used `/newpath`

Reply:
I did change it locally but somehow got a conflict and didn't update. Not sure how this happened but will be updated in the next diff.